kotsu/run.py

```diff
@@ -36,8 +36,11 @@ def run(
         skip_if_prior_result: Flag, if True then will not run validation-model combinations
             that are found in the results at given results_path. If False then all combinations
             will be ran and any prior results in results_path will be completely overwritten.
-        artefacts_store_directory: A directory path or URI location to store extra output
-            artefacts of the validations and models.
+        artefacts_store_directory: A directory path or URI location to store extra output artefacts
+            of the validations and models.
+            If not None, then validations will be passed, via the same kwarg name
+            `artefacts_store_directory`, this arg value but with validation ID and model ID
```

Comment:
clarity/reword needed: my understanding is: If not None then everything happens just as when it is None, but the artefacts are also stored in that store directory?

Reply:
Is this looks like it is a comment on out of date code? If you look at 42 `run.py` in changes it is different the the text that is highlighted by this comment?